app/org/sagebionetworks/bridge/BridgeUtils.java

```diff
@@ -66,6 +68,15 @@
     // "request context" object into every method of every class.
     private static final ThreadLocal<String> REQUEST_ID_THREAD_LOCAL = ThreadLocal.withInitial(() -> null);
 
+    public static Tuple<String> parseAutoEventValue(String automaticEventValue) {
+        // Will property split something like "custom:myEvent:P3W"
+        int lastIndex = automaticEventValue.lastIndexOf(":P");
+        if (lastIndex == -1) {
+            return new Tuple<>(ActivityEventObjectType.ENROLLMENT.name().toLowerCase(), automaticEventValue); 
+        }
```

Comment:
We shouldn't default to enrollment. mPower 2.0 configs are going to change anyway, so back-compat is a non-issue. Adding a special case for enrollment just adds complexity and confusion.

If we absolutely need a default for some reason, that default should be activities_retrieved.

Reply:
Okay we don't default any longer, and the validator will return validation errors if the string is something along the lines of "P3D". You need to add either a system event or a custom event key to trigger off of. The existing mPower 2.0 config won't be valid after deployment.